### DISCUSSION TITLE
Script. list of "model"

### DISCUSSION THREAD

**Discussion by romarioZ1000** [netbox-community/netbox#11242] [Q&A]
Hi all

i have:

vlan_id = ObjectVar(
        model = VLAN,
        label = 'VLAN (ID)',
    )

and returned me vlan name

but i need vlan ID.
and
where can i see the whole list of "model"?

Thank all

**Comment by ziggekatten** [comment]
Try:

vlan_id = ObjectVar(
model = VLAN,
label = 'VLAN (ID)',
display = vid
)

**Reply by romarioZ1000** [reply]
returned:

<class 'NameError'>

name 'vid' is not defined
